gapir/service/service.proto

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package service;
```

Comment:
@ben-clayton This is causing duplicated registered proto name with `gapis/service/service.proto`, which has  `PingRequest/Response etc`, should I rename it to `gapir_service` or `GapirService` or any other suggestions?

Reply:
As we discussed, I suggest `package gapir` and `service Service`